data/data/aws/vpc/sg-master.tf

```diff
@@ -80,6 +80,46 @@ resource "aws_security_group_rule" "master_ingress_vxlan_from_worker" {
   to_port   = 4789
 }
 
+resource "aws_security_group_rule" "master_ingress_geneve" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.master.id}"
+
+  protocol  = "udp"
+  from_port = 6081
+  to_port   = 6081
+  self      = true
+}
+
+resource "aws_security_group_rule" "master_ingress_geneve_from_worker" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.master.id}"
+  source_security_group_id = "${aws_security_group.worker.id}"
+
+  protocol  = "udp"
+  from_port = 6081
+  to_port   = 6081
+}
+
+resource "aws_security_group_rule" "master_ingress_ovndb" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.master.id}"
+
+  protocol  = "udp"
```

Comment:
"tcp"
which is also to say,
/hold
until this is tested

Reply:
@danwinship Yeah, good catch. Will you be able to test these rules while doing your ovn bringup/cleanup?